### ISSUE TITLE
No visible focus indicator when focused on a selected find filter

### ISSUE THREAD

**Issue by rzhao271** [microsoft/vscode#165386]
1. Ctrl+F to open the editor find widget. Focus should be on the find input box.
2. Enable case-sensitive search.
3. Focus to the case-sensitive search button.
4. :bug: Keyboard focus is on that element, but it's not clear that keyboard focus is on that element.

![Screencap showing steps outlined above](https://user-images.githubusercontent.com/7199958/199773288-e095dce4-44c1-4c79-b7a8-a4326ff4716f.gif)

**Comment by jNullj** [comment]
works fine on my machine with Light+ and Dark+ theme, what theme are you using?
![Peek 2022-11-04 00-28](https://user-images.githubusercontent.com/15849761/199844635-f62f55f3-38df-4144-b541-2736ea0ed1bf.gif)
![Peek 2022-11-04 00-29](https://user-images.githubusercontent.com/15849761/199844734-15701b21-c046-48df-9aaf-613666ec65fc.gif)

**Comment by rzhao271** [comment]
Thanks! I guess the problem depends on the theme.
The problem seems to occur on at least Quiet Light, Solarized Light, and Monokai.

**Comment by jNullj** [comment]
> Thanks! I guess the problem depends on the theme. The problem seems to occur on at least Quiet Light, Solarized Light, and Monokai.

Can confirm the the issue in those themes on my windows machine.
I never sent a pull request to vscode but that seems easy to solve, the question is what highlight color would fit best each theme, i might take a look later unless there are some rules about how to set some of the theme colors...

the colors effecting this seem to be
"focusBorder" and "inputOption.activeBorder"
I would love to offer some new colors later today/tomorrow and pull request them

**Comment by jNullj** [comment]
I propose giving a bit of contrast to the "focusBorder"
I think that focus border should have a bit of contrast from background as this is marking elements the user needs to spot quickly, even in themes with less contrast light quiet light i think this is best to increase contrast in those areas, this is only showing for elements put in focus and wont change the feeling of the theme by much.

Here are the new colors i propose and will commit and pull request later if those look acceptable

For Quiet Light:
![quiet light](https://user-images.githubusercontent.com/15849761/200114260-6c6cc282-c0f1-48a0-a056-9c20ad239176.gif)
For Solarized Light:
![solarized light](https://user-images.githubusercontent.com/15849761/200114588-c0754576-f7a1-41a7-9f29-9c8e06bed6dd.gif)
For Monokai:
![monokai](https://user-images.githubusercontent.com/15849761/200114723-4b3f16a2-d28c-4f5b-82b7-1926a9ddd8f7.gif)
